src/applications/disability-benefits/all-claims/tests/validations.unit.spec.js

```diff
@@ -545,7 +545,7 @@ describe('526 All Claims validations', () => {
       const errors = { addError: sinon.spy() };
       const dob = '2000-01-01';
       // Add 1 extra day to ensure we're after 13th birthday
-      const age = formatDate(add(new Date(dob), { years: 13, days: 1 }));
+      const age = formatDate(add(new Date(dob), { years: 13, days: 2 })); // This test was failing with days: 1
```

Comment:
Nit: The comment above should be updated to 2 extra days

Reply:
This has been updated